### DISCUSSION TITLE
Is there any alternative to mutateFormDataBeforeCreate for form package?

### DISCUSSION THREAD

**Discussion by ArtMin96** [filamentphp/filament#6304] [Help] [answered]
Is there any alternative to [mutateFormDataBeforeCreate](https://filamentphp.com/docs/2.x/admin/resources/creating-records#customizing-data-before-saving) for form package?

**Comment by danharrin** [comment, marked as answer]
If you're using the form package, you can just mutate the form data from the `getState()` array before you save it anyway. This hook is only useful in the admin panel

**Reply by ArtMin96** [reply]
Thanks
